WordPress/Classes/Services/MediaSettings.swift

```diff
@@ -3,14 +3,70 @@ import AVFoundation
 
 class MediaSettings: NSObject {
     // MARK: - Constants
+    fileprivate let imageOptimizationKey = "SavedImageOptimizationSetting"
     fileprivate let maxImageSizeKey = "SavedMaxImageSizeSetting"
+    fileprivate let imageQualityKey = "SavedImageQualitySetting"
     fileprivate let removeLocationKey = "SavedRemoveLocationSetting"
     fileprivate let maxVideoSizeKey = "SavedMaxVideoSizeSetting"
+    fileprivate let advertiseImageOptimizationKey = "SavedAdvertiseImageOptimization"
 
+    fileprivate let defaultImageOptimization = true
+    fileprivate let defaultMaxImageSize = 2000
```

Comment:
My programmer brain wants to replace this with 2048 🙈

Reply:
> I also like using "dimension" over "size" so the reader easily knows we're talking about resolution rather than file size.

@twstokes This makes me wonder if we should also update the label of this setting. It currently says `Max Image Upload Size`, which as you pointed out, might be interpreted as the file size, not dimensions.

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21981/commits/477777cbb71f90fe99b205de6a6d35543343705a.